### PR TITLE
Multi-thread Maven Build

### LINKED ISSUE #4349: Test should be executed in parallel to reduce build time
It should be possible to execute unit tests in parallel and we could introduce multi-threaded builds to further improve the build time of `activiti-cloud`.

### PR DESCRIPTION
**Pull request by LorenzoDiGiuseppe**
This PR fixes Activiti/Activiti#4349 as it configures the project for multithread builds:
- it introduces the profile `acceptance-tests` with activation on property `skipAcceptanceTests=false` which is the default value to exclude `serenity-maven-plugin` from the main aggregator since it was causing random failures in the following goals: `default-testCompile`, `test`, `integration-test`
- the Unit Tests are executed in parallel by classes, meaning that UT methods are executed sequentially inside a class avoiding conflicts
- nothing is changed for Integration Tests since they cannot be executed in parallel unless defining `@ResourceLocks` or `@Isolated` on test classes to avoid race-conditions (it would not improve the performances in the end, see: Activiti/activiti-cloud#1073)
- Integration Tests belonging to different modules are executed in parallel, so it has been required removing static service ports where defined to avoid errors
- `license-maven-plugin` has been upgraded to version 2.1.0 that includes the changes from the 2.0.1-alfresco.2 and it marks the goal `add-third-party` as `threadsafe`.

__NOTE__: the plugin `assertj-assertions-generator-maven-plugin` is not marked as `threadsafe`, but the [Mojo execution](https://github.com/assertj/assertj-assertions-generator-maven-plugin/blob/ae656e8caf99c7c27f725be8fa6b68b334b27ba1/src/main/java/org/assertj/maven/AssertJAssertionsGeneratorMojo.java#L209-L213) seems to be safe enough to use.

### CODE REVIEW

**Review by andrea-ligios — APPROVED** [review]
LGTM!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ jobs:
 
       - name: Maven Build and Test
         shell: bash
-        run: mvn -DnoAcceptanceTests=true install ${{ env.MAVEN_CLI_OPTS}}
+        run: mvn -DnoAcceptanceTests=true -DunitTests.parallel=true -T 1C install ${{ env.MAVEN_CLI_OPTS}}
```
Comment (erdemedeiros) [inline .github/workflows/main.yml]: I know that this was not introduced by this PR,  but what do you think about renaming `noAcceptanceTests` to `skipAcceptanceTests`?

File: activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle-it/src/test/resources/application.properties [hunk 2]
```diff
@@ -7,7 +7,7 @@ spring.jmx.enabled=false
 spring.rabbitmq.host=localhost
 
 # Shared H2 database instance
-spring.datasource.url=jdbc:h2:tcp://localhost:9090/mem:mydb
+# spring.datasource.url=jdbc:h2:tcp://localhost:9090/mem:mydb
```
Comment (erdemedeiros) [inline activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle-it/src/test/resources/application.properties]: Please, remove commented code.

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
Great job, @LorenzoDiGiuseppe !
There is one small change needed and one optional.

File: activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle-it/src/test/resources/application.properties (hunk 2 above)
Comment (LorenzoDiGiuseppe) [reply activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle-it/src/test/resources/application.properties]: Done, thank you.

File: .github/workflows/main.yml (hunk 1 above)
Comment (LorenzoDiGiuseppe) [reply .github/workflows/main.yml]: Done, thank you.

**Review by erdemedeiros — APPROVED** [review]
LGTM!

**Pull request merged**